app/assets/javascripts/discourse/app/components/sidebar/hamburger-dropdown.hbs

```diff
@@ -3,7 +3,7 @@
     <div class="panel-body">
       <div class="panel-body-contents">
         <div class="sidebar-hamburger-dropdown">
-          <Sidebar::User::Sections @collapsableSections={{false}}/>
+          <Sidebar::Sections @currentUser={{this.currentUser}} @collapsableSections={{false}} />
```

Comment:
Is it necessary to pass down the currentUser?

Reply:
Yes, if I want to keep `<Sidebar::Sections>` as a template only component. I realised this is one of the drawbacks of converting `currentUser` to a service... because each component has to inject the service. I'm not sure if there is a better way here? cc @davidtaylorhq

Reply:
And what do we win to keep it as a template only component? besides few loc?

Reply:
technically performance because template only components are much faster. On the flip side, is passing down a service from another component bad? I'm not really sure to be honest.

Reply:
Yes also don't know, but passing down `currentUser` is kinda odd in our codebase for sure. Anyways, don't want to block you on this, we can change this later if someone has more hindsights.

Reply:
Yup we can definitely discuss further and see what David thinks as well.